### PR TITLE
EZP-29744: Implement permissions for Content/Create in Sub-items table

### PR DESCRIPTION
**Pull request by mikadamczyk**
| Question      | Answer
| ------------- | ---
| Tickets       | [https://jira.ez.no/browse/EZP-29744](https://jira.ez.no/browse/EZP-29744)
| Bug fix?      |no
| New feature?  | yes
| BC breaks?    |no
| Tests pass?   | yes
| Doc needed?   |no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Requires: [https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/116](https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/116)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Comment by mikadamczyk** [comment]
backend part is ready to review

File: src/bundle/Resources/views/content/locationview.html.twig [hunk 1]
```diff
@@ -68,7 +68,8 @@
                             <h2 class="ez-fieldgroup__name">{{ 'content.view.subitems'|trans|desc('Sub-items') }}</h2>
                             <div class="ez-sil" 
                                 data-location="{{ location.id }}" 
-                                data-items="{{ subitems_module.items }}" 
+                                data-mfu-can-create="{{ content_create_permissions_for_mfu|json_encode() }}"
```
Comment (tischsoic) [inline src/bundle/Resources/views/content/locationview.html.twig]: Does `data-mfu-can-create` store `contentCreatePermissionsConfig`, which is an object? If so, I would rename this data-attribute to e.g. `data-mfu-create-permissions-config`, which for me is more precise.

Comment (sunpietro) [reply src/bundle/Resources/views/content/locationview.html.twig]: Yes, it does

File: src/bundle/Controller/ContentViewController.php [hunk 2]
```diff
@@ -449,4 +469,41 @@ private function supplyIsLocationBookmarked(ContentView $view): void
...
     }
+
+    /**
+     * @param \eZ\Publish\Core\MVC\Symfony\View\ContentView $view
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     */
+    private function supplyContentCreatePermissionsForMFU(ContentView $view): void
```
Comment (webhdx) [inline src/bundle/Controller/ContentViewController.php]: You are hiding quite complicated logic in controller. Better place for your code is in `\EzSystems\EzPlatformAdminUi\UI\Config\Provider\Module\SubItemsList`

**Pull request merged**